rpc/openapi/openapi.yaml

```diff
@@ -911,6 +911,30 @@ paths:
             application/json:
               schema:
                 $ref: "#/components/schemas/ErrorResponse"
+  /genesis_chunked:
+     get:
+      summary: Get Genesis Chunked
+      operationId: genesis_chunked
+      tags:
+        - Info
+      description: |
+        Get genesis chunked.
+
+        Upon success, the `Cache-Control` header will be set with the default
+        maximum age.
+      responses:
+        "200":
+          description: Genesis chunked results.
+          content:
+            application/json:
+              schema:
+                $ref: "#/components/schemas/GenesisChunkedResponse"
```

Comment:
Does this component already exist somewhere?

Reply:
No, this schema still needs to be added to the bottom of the doc.

Reply:
Yes, just added this one in